concent_api/concent_api/system_check.py

```diff
@@ -280,6 +282,22 @@ def create_error_42_verifier_download_chunk_size_has_wrong_value(verifier_downlo
     )
 
 
+def create_error_43_signing_service_public_key_is_missing():
+    return Error(
+        'SIGNING_SERVICE_PUBLIC_KEY is not defined',
+        hint='SIGNING_SERVICE_PUBLIC_KEY should be defined in settings if "middleman" feature is enabled.',
+        id='concent.E040',
```

Comment:
Number of error is not the same as number in function name. Also this error number exists.

Reply:
Done